### DISCUSSION TITLE
Filter commit graph to only show search results?

### DISCUSSION THREAD

**Discussion by ankitson** [gitkraken/vscode-gitlens#2897] [Q&A]
When I search for something in the Commit Graph view, it only dims the non-matching commits instead of hiding them. When searching for needles in haystacks this still makes it difficult to find the ones I need. Is there a way to only show commits matching the search filter either using the commit graph view or some other view?

**Comment by eg-ma** [comment]
This would be a good feature to have. I'm in the same situation.
